packages/strapi-design-system/src/Select/Select.js

```diff
@@ -53,7 +53,8 @@ export const Select = ({
 
   const labelId = `${generatedId}-label`;
   const contentId = `${generatedId}-content`;
-  const ariaDescribedBy = error ? `${generatedId}-error` : hint ? `${generatedId}-hint` : undefined;
+  const ariaDescribedBy =
+    error && typeof error === 'string' ? `${generatedId}-error` : hint ? `${generatedId}-hint` : undefined;
```

Comment:
These nested ternaries aren't easy to read. IMO you can do something like that

```suggestion
  const hasStringError = typeof error === 'string';
  const errorMessageId = hasStringError ? `${generatedId}-error` : undefined;
  const hintMessageId = hint ? `${generatedId}-hint` : undefined;
  
  const ariaDescribedBy = errorMessageId || hintMessageId;
```

Reply:
I like it, thanks a lot!